components/word-visualiser.tsx

```diff
@@ -58,7 +58,10 @@ const WordVisualiser = (): React.ReactElement | undefined => {
 		}
 
 		if (state.word.endTime !== undefined && !state.word.match) {
-			return 'Typo';
+			let output = 'Typo';
+			if (state.typoText != undefined && state.typoText != ' ')
+			output += `: ${state.typoText}`;
+			return output;
```

Comment:
This doesn't follow the code style conventions; you'll need to add the missing braces on the if statement, add in the correct newline separators, and use strict equality checks:

```suggestion
			let output = 'Typo';

			if (state.typoText !== undefined && state.typoText !== ' ') {
				output += `: ${state.typoText}`;
			}

			return output;
```

Reply:
`state.typoText` is also missing from the `useMemo` dependency array below.